boards/arm/lpcxpresso51u68/lpcxpresso51u68.dts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2021 metraTec GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_lpc51u68.dtsi>
+#include <dt-bindings/pwm/pwm.h>
+
+/ {
+	model = "NXP LPCXPRESSO51U68 board";
+	compatible = "nxp,lpc51xxx","nxp,lpc";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &flexcomm0;
+		zephyr,shell-uart = &flexcomm0;
+	};
+
+	aliases {
+		led0 = &blue_led;
+		led1 = &green_led;
+		led2 = &red_led;
+		pwm-0 = &sc_timer;
+		pwm-led0 = &green_pwm_led;
+		green-pwm-led = &green_pwm_led;
+		sw0 = &btn_wk;
+		sw1 = &isp0_button;
+		sw2 = &isp1_button;
+		usart-0 = &flexcomm0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led: led_0 {
+			gpios = <&gpio1 9 0>;
+			label = "Blue LED";
+		};
+		green_led: led_1 {
+			gpios = <&gpio1 10 0>;
+			label = "Green LED";
+		};
+		red_led: led_2 {
+			gpios = <&gpio0 29 0>;
+			label = "Red LED";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		green_pwm_led: green_pwm_led {
+			pwms = <&sc_timer 4 PWM_POLARITY_NORMAL>;
```

Comment:
This might need to be updated per changes to PWM. This might be causing the CI failures.